src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRental = 40;
+  const basicDiscount = 20;
+  const maxDiscount = 50;
+  const basicSum = days * dailyRental;
+
+  if (days >= 7) {
+    return basicSum - maxDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
This second check is really redundant, you already checked that days count is not bigger than 7
```suggestion
  if (days >= 3) {
```